helloworld/README.md

```diff
@@ -1,10 +1,15 @@
-### Steps to run this sample:
-1) You need a Temporal service running. See details in README.md
-2) Run the following command to start the worker
+## HelloWorld! Temporal Go SDK sample
+
+To use this sample, first [Install and run the Temporal server](https://docs.temporal.io/docs/install-temporal-server).
+
+Run the Workflow:
+
 ```
-go run helloworld/worker/main.go
+go run helloworld/starter/main.go
```

Comment:
Why flip the order? To show that starter will be blocked until the worker starts?

Reply:
Yes.